### ISSUE TITLE
ATPL templating engine is not working as expected

### ISSUE THREAD

**Issue by slavafomin** [balderdashy/sails#2898]
# Steps to reproduce:

1. Generate new Sails project
2. Install ATPL via: `npm install --save atpl`
3. Change views config to use ATPL
4. Add `homepage.atpl`
5. Lift the app and try to load the page

# Expected behavior

ATPL template is rendered and returned.

# Actual behavior

Server throws an exception: `Error: ENOENT, no such file or directory '/home/sfomin/sails-project/views/home/sfomin/sails-project/views/homepage.atpl'`

# Thoughts

Looks like there is some issue with paths resolving.

# Used files

```javascript
// config/views.js
module.exports.views = {
  engine: 'atpl'
};
```

```javascript
// config/routes.js
module.exports.routes = {
  '/': {
    view: 'homepage'
  }
};
```

```jinja2
{# views/homepage.atpl #}
<h1>Hello!</h1>
```

**Comment by slavafomin** [comment]
I was actually writing an article and my point was to show how it is easy to change the templating engine used with Sails ))

If it's possible to fix this or point me in the right direction I will really appreciate it, thanks!

**Comment by sailsbot** [comment]
Thanks for posting, @slavafomin.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
